### PR TITLE
remove v0.7.0 warning

### PR DESCRIPTION
**Pull request by eitanturok**
## Description of changes:

remove part of the warning messages about v0.7.0 of streaming.

## Merge Checklist:
_Put an `x` without space in the boxes that apply. If you are unsure about any checklist, please don't hesitate to ask. We are here to help! This is simply a reminder of what we are going to look for before merging your pull request._

### General
- [x] I have read the [contributor guidelines](https://github.com/mosaicml/streaming/blob/main/CONTRIBUTING.md)
- [ ] This is a documentation change or typo fix. If so, skip the rest of this checklist.
- [x] I certify that the changes I am introducing will be backward compatible, and I have discussed concerns about this, if any, with the MosaicML team.
- [x] I have updated any necessary documentation, including [README](https://github.com/mosaicml/streaming/blob/main/README.md) and [API docs](https://github.com/mosaicml/streaming/tree/main/docs) (if appropriate).

### Tests
- [ ] I ran `pre-commit` on my change. (check out the `pre-commit` section of [prerequisites](https://github.com/mosaicml/streaming/blob/main/CONTRIBUTING.md#prerequisites))
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate).
- [ ] I ran the tests locally to make sure it pass. (check out [testing](https://github.com/mosaicml/streaming/blob/main/CONTRIBUTING.md#submitting-a-contribution))
- [ ] I have added unit and/or integration tests as appropriate to ensure backward compatibility of the changes.

### CODE REVIEW

File: streaming/base/dataset.py [hunk 1]
```diff
@@ -415,9 +415,7 @@ def __init__(self,
                           f'predownload to at-least batch_size.')
         elif self.predownload is None:
             logger.warning(f'Because `predownload` was not specified, it will default to ' +
-                           f'8*batch_size if batch_size is not None, otherwise 64. Prior to ' +
-                           f'Streaming v0.7.0, `predownload` defaulted to ' +
-                           f'max(batch_size, 256 * batch_size // num_canonical_nodes).')
+                           f'8*batch_size if batch_size is not None, otherwise 64.')
```
Comment (mvpatel2000) [inline streaming/base/dataset.py:418]: we shoudl lower to `logger.info`

File: streaming/base/dataset.py [hunk 2]
```diff
@@ -671,8 +669,7 @@ def _set_shuffle_block_size(self, world: World):
             if not world.worker_of_rank:
                 logger.warning(f'Because `shuffle_block_size` was not specified, it will ' +
                                f'default to max(4_000_000 // num_canonical_nodes, 1 << 18) if ' +
-                               f'num_canonical_nodes is not None, otherwise 262144. Prior to ' +
-                               f'Streaming v0.7.0, `shuffle_block_size` defaulted to 262144.')
+                               f'num_canonical_nodes is not None, otherwise 262144.')
```
Comment (mvpatel2000) [inline streaming/base/dataset.py:672]: we should lower to `logger.info`

File: streaming/base/dataset.py [hunk 3]
```diff
@@ -700,9 +697,7 @@ def _resume(self, world: World, epoch: int) -> tuple[int, int]:
                         logger.warning(
                             f'Because `num_canonical_nodes` was not specified, and ' +
                             f'`shuffle_algo` is {self.shuffle_algo}, it will default to ' +
-                            f'be equal to physical nodes. Prior to Streaming ' +
-                            f'v0.7.0, `num_canonical_nodes` defaulted to 64 * physical ' +
-                            f'nodes.')
+                            f'be equal to physical nodes.')
```
Comment (mvpatel2000) [inline streaming/base/dataset.py:700]: we should lower to `logger.info`

File: streaming/base/dataset.py [hunk 4]
```diff
@@ -723,9 +718,7 @@ def _resume(self, world: World, epoch: int) -> tuple[int, int]:
...
```
Comment (mvpatel2000) [inline streaming/base/dataset.py:721]: we should lower to `logger.info`

**Review by mvpatel2000 — APPROVED** [review]

**Comment by snarayan21** [comment]
Closing in favor of #799

**Pull request closed without merging**